### PR TITLE
Werkzeug 2.2.2

### PR DESCRIPTION
**Pull request by SC426**
Jira Ticket: https://anaconda.atlassian.net/browse/PKG-596
Upstream Repo: https://github.com/pallets/werkzeug/tree/2.2.2
Changelog: https://github.com/pallets/werkzeug/tree/2.2.2/docs

This will cause some breaking changes, and is part of a larger ticket the apprentices are working on.

### CODE REVIEW

File: abs.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+channels:
```
Comment (katietz) [inline abs.yaml:1]: please remove this channel part from abs.yaml

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: b863f8ff057c522164b6067c9e28b041161b4be5ba4d0daceeaa50a163822d3c
+  sha256: 7ea2d48322cc7c0f8b3a215ed73eabd7b5d75d0b50e31ab006286ccff9e00b8f
 
 build:
-  noarch: python
+  skip: True #[py<37]
```
Comment (katietz) [inline recipe/meta.yaml:13]: please add a space before, and one after the '#'. Selectors should have the pattern '  # [<selector-condition>]'

**Review by katietz — CHANGES_REQUESTED** [review]
ok, with those 2 changes

**Review by varlackc — COMMENTED** [review]
As soon as we fix the issue with the skip we should be good to approve the package.

File: abs.yaml (hunk 1 above)
Comment (marcoesters) [reply abs.yaml:1]: The channel part is important for now because it is part of a dependency chain. The file will be removed before merging.

**Comment by SC426** [comment]
> As soon as we fix the issue with the skip we should be good to approve the package.

File: recipe/meta.yaml (hunk 2 above)
Comment (SC426) [reply recipe/meta.yaml]: This has been fixed

**Review by skupr-anaconda — APPROVED** [review]

**Comment by SC426** [comment]
> As soon as we fix the issue with the skip we should be good to approve the package.

This has been fixed.

**Review by marcoesters — APPROVED** [review]
LGTM

Wait until all packages that depend on werkzeug have been updated before merging though. We have to merge/upload this concurrently with `flask` and `airflow`

**Pull request merged**